pages.es/common/lstopo.md

```diff
@@ -0,0 +1,24 @@
+# lstopo
+
+> Muestra la topología de hardware del sistema.
+> Más información: <https://manned.org/lstopo>.
+
+- Muestra la topología resumida del sistema en una ventana gráfica (o imprime en consola si no se dispone de pantalla gráfica):
+
+`lstopo`
+
+- Muestra la topología completa del sistema sin resúmenes:
+
+`lstopo --no-factorize`
+
+- Muestra la topología resumida del sistema sólo con índices [físicos] (es decir, tal y como la ve el sistema operativo):
```

Comment:
```suggestion
- Muestra la topología resumida del sistema sólo con índices físicos (es decir, tal y como la ve el sistema operativo):
```

At the top of my head, I cannot think of a word suitable for a short option mnemonic. So, I opt for not having it all, what do you think @kant?

Reply:
@tricantivu I tried the best, but I agree in a whole with your point of view